test/test_ITF1788/ieee1788-exceptions.jl

```diff
@@ -0,0 +1,12 @@
+@testset "exceptions" begin
+
+    @test_broken @interval("[+infinity]") === emptyinterval()
+
+    @test interval(+Inf,-Inf) === emptyinterval()
```

Comment:
yes ITF1788.jl has a `test_warn` option to choose whether to test or not for warnings. The issue was that `@test_logs` (the macro to test for warnings) doesn't work with `@test_broken` and would need to use `@test_skip` to avoid exceptions during the tests. Since those tests with warnings were very few, I can readd those and change manually broken to skip